README.md

```diff
@@ -1,11 +1,51 @@
 # cNVMe
 An NVMe Device Simulation Library.
 
-This is a WIP simulation of the protocol-layer of an NVMe Memory Device. The goal is to be able to send NVMe commands down and get full protocol responses. The simulation includes a register level with basic PCIe Controller Registers and simplified Doorbell Registers. It is really cool!
+This is a WIP simulation of the protocol-layer of an NVMe Memory Device. The goal is to be able to send NVMe commands down and get full protocol responses. The simulation includes a register level with basic PCIe Controller Registers and simplified Doorbell Registers. It also contains a "Driver" with some abstractions to make it much easier to send commands to the underlying controller. cNVMe is really cool!
+
+## Supported NVMe Specification Commands
+### Admin Commands
+* Create IO Completion Queue
+* Create IO Submission Queue
+* Delete IO Completion Queue
+* Delete IO Submission Queue
+* Delete IO Submission Queue
```

Comment:
Do you want to maybe put references to NVMe spec sections with corresponding commands??

Reply:
Not really, since the reference changes spec-to-spec and is really easy to find via the table of contents in the actual specs.